src/styles/settings.scss

```diff
@@ -32,7 +97,7 @@ $photo-bg-color: #f4f5f7;
 
 $box-border-color: #e1e1e1;
 
-$stars-color: rgb(42, 42, 42);
+$stars-color: #2a2a2a;
```

Comment:
Zmienne z bezpośrednimi kolorami są zdefiniowane w górnej cześci pliku, jako nie do użytku, a tą zmienną możemy użyć bezpośrednio?

Reply:
widzialam ze jest ta zmienna uzywana w PromotedProductBox.module.scss

Reply:
No ok, ale jak to ma się to wszystkich zmian któe wprwoadzałas w ramach tego zadadnia?

Reply:
ale przeciez te kolory sa w wiekszosci uzywane dotyczy tez innych np text-color, primary

Reply:
dalej line-color....

Reply:
$stars-color powinin być ustawiony na $darkGray a nie powinno mieć zapisany kolor tak samo inne zmienne któe pojawiły się po mergu, mają przypisany bezpośrednio kolor zmiast bazować na zmiennych